.gitignore

```diff
@@ -5,7 +5,7 @@ src/bitcoin
 src/zcashd
 src/zcash-cli
 src/zcash-gtest
-src/bitcoin-tx
+src/zcash-tx
 src/test/test_bitcoin
```

Comment:
(because it would require having free space on the CI, which isn't going to happen anytime soon.)